### PR TITLE
feat(response-ratelimiting): add support for Redis SSL

### PR DESCRIPTION
**Pull request by vm-001**
### Summary

Add corresponding docs for the PR https://github.com/Kong/kong/pull/8595

### CODE REVIEW

**Comment by vm-001** [comment]
Target release version 3.1

File: app/_hub/kong-inc/response-ratelimiting/_index.md [hunk 1]
```diff
@@ -172,6 +172,26 @@ params:
...
+      datatype: boolean
+      description: |
+        When using the `redis` policy, this property specifies if SSL is used to connect to the Redis server.
+    - name: redis_ssl_verify
+      minimum_version: "3.1.x"
+      required: true
+      default: '`false`'
+      datatype: boolean
+      description: |
+        When using the `redis` policy with `redis_ssl` set to `true`, this property specifies it server SSL certificate is validated. Note that you need to configure the lua_ssl_trusted_certificate to specify the CA (or server) certificate used by your Redis server. You may also need to configure lua_ssl_verify_depth accordingly.
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/response-ratelimiting/_index.md:188]:
```suggestion
        When using the `redis` policy with `redis_ssl` set to `true`, this property specifies if the server SSL certificate is validated. Note that you need to configure the `lua_ssl_trusted_certificate` to specify the CA (or server) certificate used by your Redis server. You may also need to configure `lua_ssl_verify_depth` accordingly.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Just one small edit, thanks for submitting this!

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**